app/controllers/settings_controller.py

```diff
@@ -444,6 +445,19 @@ def _update_view_from_model(self) -> None:
             self.settings.todds_overwrite
         )
 
+        # themes tab
+        if self.settings.enable_themes:
+            self.settings_dialog.enable_themes_checkbox.setChecked(True)
+        # get theme names
+        current_theme_name = self.settings.theme
+        current_index = self.settings_dialog.themes_combobox.findText(
+            current_theme_name
+        )
+        if current_index != -1:
```

Comment:
Why force the index to be 0 if the current_index is -1?

Worst case this causes a crash if there is no index 0, best case it picks a theme that the user did not intend to pick. IMO for debugging purposes and just user clairity, it'd just be better to leave it as -1 and down the line handle it and understand that -1 means the selection is blank.

Reply:
This is for a reason if a theme other then Default is set and for some reason it goes missing then instead of showing blank selection it alsways points to Rimpy which is default theme
this has also been addressed in multiple places as a failsafe to avoid the app from launching or crashing just beause of theme 
i could not crash the app due to themes in my testings feel free to crash it :-)

Reply:
It does crash if you delete the RimPy theme from the themes folder, albeit the error happens upstream before this even has a chance to error.

Though the user is instructed to put their personal themes under the config folder, I think it's reasonable to say that the app shouldn't crash if the user clears out their themes folder accidentally, especially since it is just named "themes" and a lot of other apps have users just place themes into a themes.

For this behavior, it should default to no theme selected, and fall back to system theme instead of using a qss stylesheet.

Reply:
ok foxed